app/controllers/agent/EnterClientsUTRController.scala

```diff
@@ -90,11 +93,13 @@ class EnterClientsUTRController @Inject()(enterClientsUTR: EnterClientsUTR,
                     SessionKeys.clientNino -> nino,
                     SessionKeys.clientUTR -> validUTR
                   ) ++ firstName.map(SessionKeys.clientFirstName -> _) ++ lastName.map(SessionKeys.clientLastName -> _)
```

Comment:
:pushpin: - this might be not a good idea to rely on SessionKeys, overall tendency would be to remove/decommission these from the code base as a redundant concept;

Reply:
I think this data we are still storing in cookies. Maybe we could make a separate ticket to migrate this to mongo since this seems to be outside the scope of the ticket?

Reply:
oh .. there must a ticket already for this but its not directly related:
https://jira.tools.tax.service.gov.uk/browse/MISUV-6408